### DISCUSSION TITLE
Status of char8_t support?

### DISCUSSION THREAD

**Discussion by kamrann** [nlohmann/json#2334] [Q&A] [answered]
Changelog for 3.8.0 lists `C++20: char8_t`, but from what I can see, it's still not possible to compile when using a `char8_t`-based string (eg. `std::u8string`) as `basic_json`'s `StringType`. From what I can see, the library is still using hard-coded `char` in many places.
What's the status on this?

**Comment by nlohmann** [comment, marked as answer]
The ChangeLog is generated automatically and lists all closed issues since the last release. #1914 and #2097 were not worked on as support for C++20 types is not a priority. In #2337, a feature request is described. Please follow up there.
